src/pytest_workflow/workflow.py

```diff
@@ -137,6 +138,9 @@ def wait(self, timeout_secs: Optional[float] = None,
             # workflow
             pass
 
+    def matching_exitcode(self) -> bool:
```

Comment:
Please put a docstring here and also a comment why this feature is needed. We violate the modular design of pytest-workflow here by combining some of the test code with the workflow running code. Normally these would be separate. So a comment on this decision is needed.